content/platframe-default-starter.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Platframe default starter"
+github: https://github.com/platframe/platframe
+author: platframe
+demo: https://default.platframe.com/
+date: 2020-06-03
+ssg:
+    - Platframe
```

Comment:
Platframe will need to be added as a ssg taxonomy

Reply:
Fixed